### PR TITLE
Fix inbox telemetry 

### PR DESCRIPTION
**Pull request by ryanlai2**
Inbox builds shouldn't have Onnxruntime telemetry reporting "isRedist" as true.

This change makes it so that inbox builds report "isRedist" as false.

If onnxruntime_WINML_NAMESPACE_OVERRIDE  equals "Windows" then it should be building inbox. I am following the same format as here : https://github.com/microsoft/onnxruntime/blob/75d994f1940444022bc22127c87f56636df884a6/cmake/winml.cmake#L210

### CODE REVIEW

File: cmake/onnxruntime_common.cmake [hunk 1]
```diff
@@ -135,3 +135,7 @@ if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
     target_link_libraries(onnxruntime_common rt)
   endif()
 endif()
+
+if (onnxruntime_WINML_NAMESPACE_OVERRIDE STREQUAL "Windows")
```
Comment (zhangxiang1993) [inline cmake/onnxruntime_common.cmake:139]: >onnxruntime_WINML_NAMESPACE_OVERRIDE [](start = 4, length = 36)

It looks we don't override the namespace in WAI-nightly-build.

Comment (shuweihsu) [reply cmake/onnxruntime_common.cmake:139]: Where can I find this "Windows" override?

Comment (ryanlai2) [reply cmake/onnxruntime_common.cmake:139]: It is set here : https://microsoft.visualstudio.com/windowsai/_git/WindowsAI?path=%2FCMakeLists.txt&version=GBmaster&line=176&lineEnd=177&lineStartColumn=1&lineEndColumn=1&lineStyle=plain

Comment (ryanlai2) [reply cmake/onnxruntime_common.cmake:139]: image

Comment (ryanlai2) [reply cmake/onnxruntime_common.cmake:139]: From my WAI build run : 
image

**Review by shuweihsu — APPROVED** [review]

**Review by angsabrina — APPROVED** [review]

**Review by zhangxiang1993 — APPROVED** [review]
:shipit:

**Pull request merged**